content/en/docs/setup/production-environment/container-runtimes.md

```diff
@@ -45,52 +45,31 @@ check the documentation for that version.
 {{< /note >}}
 
 <!-- body -->
-## Install and configure prerequisites
+## Network configuration
 
-The following steps apply common settings for Kubernetes nodes on Linux.
+By default, the Linux kernel does not allow IPv4 packets to be routed
+between interfaces. Most Kubernetes cluster networking implementations
+will change this setting (if needed), but some might expect the
+administrator to do it for them. (Some might also expect other sysctl
+parameters to be set, kernel modules to be loaded, etc; consult the
+documentation for your specific network implementation.)
 
-You can skip a particular setting if you're certain you don't need it.
-
-For more information, see
-[Network Plugin Requirements](/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins/#network-plugin-requirements)
-or the documentation for your specific container runtime.
```

Comment:
Is this no longer helpful? The advice to read your network plugin's docs seems valid.

Reply:
yeah, I kept that ("consult the documentation for your specific network implementation"). The internal link didn't have anything useful other than a second recommendation to read your network plugin's docs.